lib/google_crawler_web/controllers/registration_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule GoogleCrawlerWeb.RegistrationController do
+  use GoogleCrawlerWeb, :controller
+
+  alias GoogleCrawler.Accounts
+  alias GoogleCrawler.Accounts.User
+
+  def new(conn, _params) do
+    changeset = User.changeset(%User{})
```

Comment:
Not using the registration changeset? 🤔 I actually don't see it used anywhere yet

Reply:
Currently, It is called in

```
def create_user(attrs \\ %{}) do
    %User{}
    |> User.registration_changeset(attrs)
    |> Repo.insert()
  end
```

Actually maybe I can call it here too. I'll make a change 👍